steps/steps.go

```diff
@@ -82,11 +79,10 @@ func DefaultDeployStepList(isIncludeCache bool) []bitriseModels.StepListItemMode
 	return stepList
 }
 
-// DefaultDeployStepListV2 ...
-func DefaultDeployStepListV2(shouldIncludeCache bool) []bitriseModels.StepListItemModel {
+func DefaultDeployStepListV2(shouldIncludeLegacyCache bool) []bitriseModels.StepListItemModel {
 	stepList := []bitriseModels.StepListItemModel{}
 
-	if shouldIncludeCache {
+	if shouldIncludeLegacyCache {
```

Comment:
Maybe this parameter can be removed completely?

Reply:
Actually, yes. I wanted to clean this up after updating all the default workflows, but I just checked now and every call is `false` in this PR.

Reply:
Actually, no 😅 Although this parameter is unused, its pair in the "default end of workflows steps" function is still used by the Flutter scanner to add the `cache-push` step. I'll clean this up once I change the Flutter workflows.